graphql/documents/data/performer.graphql

```diff
@@ -29,4 +29,8 @@ fragment PerformerData on Performer {
     stash_id
     endpoint
   }
+  details
+  deathdate
```

Comment:
Should be `death_date`. Please fix in other graphql files as well.

I realise that this looks inconsistent with `birthdate`, but birthdate is a single (compound) word, death date is not.